src/System.Memory/tests/System.Memory.Tests.csproj

```diff
@@ -4,9 +4,20 @@
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <ProjectGuid>{15DB0DCC-68B4-4CFB-8BD2-F26BCCAF5A3F}</ProjectGuid>
+    <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netcoreapp' OR '$(TargetGroup)' == 'uap'">true</IsPartialFacadeAssembly>
+    <DefineConstants Condition="'$(IsPartialFacadeAssembly)' != 'true'">$(DefineConstants);netstandard;FEATURE_PORTABLE_SPAN</DefineConstants>
```

Comment:
These constants seem unnecessary (not used).

Reply:
Good point. I added them to keep the new tests in a single file and used IFDEFS but decided that was cumbersome so broke them up into group-specific files included based on IsPartialFacadeAssembly.

I'll remove the DefineConstants.